index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="exp"></div>
+    <div class="container">
+
+        <header>
+            
+            <nav class="top">
+                <div class="app-bar">
+                    <span>9:14</span>
+                    <div class="network-bar">
+                        <img src="Normal_Button/Tertiary/Status_Bars/Wifi.png" alt="wifi"> 
+                        <img src="Normal_Button/Tertiary/Status_Bars/Battery.png" alt="battery">
+                        <img src="Normal_Button/Tertiary/Status_Bars/Cellular_Connection.png" alt="connectivity"> 
+                    </div>
+                </div>
+                <div class="toolbar">
+                    <div class="logo"><a href="#">Emmykage</a> </div>
+                        <div class="menu-bar"><img src="Normal_button/Tertiary/Icons/Icon-Menu.png" alt="menu"></div> 
+                </div>
+
+            </nav>
+        </header>
+        <section class="headline">
+            <h1>Hey There.
+                I’m Morris</h1>
+                <h3>I am a software developer</h3>
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don’t hestiate to contact me.
+                </p>
+                <ul class="social-links"><li>
+                    
+                   
+                    <li><a href="#"> <img src="Normal_Button/Tertiary/icons/Icon-GitHub.png" alt="git"></a></li>
+                    <li><a href="#"><img src="Normal_Button/Tertiary/icons/Icon-Linkedin.png" alt="linkedin"> </a></li>
+                    <li><a href="#"><img src="Normal_Button/Tertiary/icons/Icon-Angelist.png" alt="angel"> </a></li>
+                    <li><a href="#"> <img src="Normal_Button/Tertiary/icons/Icon-Twitter.png" alt="twitter"></a></li>
+                    <li><a href="#"><img src="Normal_Button/Tertiary/icons/Icon-Medium.png" alt="medium"> </a></li>
+                   
+                </ul>
+                <div><img class="feeler" src="Normal_Button/Group.png" alt="group"></div>
```

Comment:
- [ ] Kindly remove this background image from the HTML file because it's supposed to be added in your `style.css` file as a `background-image` property you can see that [link](https://www.w3schools.com/cssref/pr_background-image.asp) to know more about it.